numpy/core/numeric.py

```diff
@@ -376,6 +376,153 @@ def extend_all(module):
             __all__.append(a)
 
 
+def count_nonzero(a, axis=None):
+    """
+    Counts the number of non-zero values in the array ``a``.
```

Comment:
nitpick: there should be a blank line here (see numpy docstring format guideline). otherwise the summary tables in the docs will not have the right content